config/routes.php

```diff
@@ -20,6 +20,7 @@
 
 use Cake\Core\Plugin;
 use Cake\Routing\RouteBuilder;
+use Cake\Routing\Route\DashedRouted;
```

Comment:
Fixed in https://github.com/cakephp/app/commit/9053056e64f6282641fe36c69d5a20ae8f53ed50